StellarDotnetSdk/Sep/Sep0006/TransferServerService.cs

```
@@ -0,0 +1,663 @@
+    {
+        return await ExecuteGetAsync<T>(endpoint, queryParams, jwt).ConfigureAwait(false);
+    }
+
+    private void HandleForbiddenResponse(string errorJson)
+    {
+        if (string.IsNullOrWhiteSpace(errorJson))
+        {
+            return;
+        }
+
+        try
+        {
+            using var document = JsonDocument.Parse(errorJson);
+            if (!document.RootElement.TryGetProperty("type", out var typeProperty))
+            {
+                return;
+            }
+
+            var type = typeProperty.GetString();
+            if (string.IsNullOrWhiteSpace(type))
+            {
+                return;
+            }
+
+            if (type == "non_interactive_customer_info_needed")
+            {
+                var response = JsonSerializer.Deserialize<CustomerInformationNeededResponse>(errorJson, JsonOptions.DefaultOptions);
+                if (response != null)
+                {
+                    throw new CustomerInformationNeededException(response);
+                }
+            }
+            else if (type == "customer_info_status")
+            {
+                var response = JsonSerializer.Deserialize<CustomerInformationStatusResponse>(errorJson, JsonOptions.DefaultOptions);
+                if (response != null)
+                {
+                    throw new CustomerInformationStatusException(response);
+                }
+            }
+            else if (type == "authentication_required")
+            {
+                throw new AuthenticationRequiredException();
+            }
+        }
+        catch (CustomerInformationNeededException)
+        {
+            throw;
+        }
+        catch (CustomerInformationStatusException)
+        {
+            throw;
+        }
+        catch (AuthenticationRequiredException)
+        {
+            throw;
+        }
+        catch
+        {
+            // If parsing fails, let the original exception propagate
```

This swallows the exception, not propagate it.